mednafen/pce_fast/vdc.cpp

```diff
@@ -407,34 +406,6 @@ DECLFW(VDC_Write)
    }
 }
 
-
-// 682 + 8 + 128 = 818.
-static INLINE void CalcStartEnd(const vdc_t *vdc, uint32 &start, uint32 &end)
```

Comment:
I noticed this part wasn't working anymore as we do the picture alignment differently now.
The Hacks for Asuka and Addams Family where there but doing nothing, that's why I brought them later in the code in a different way.
I tested all kind of games resolutions after removing this, no change at all.
Perhaps we could clean more under line 894 but I'm not sure.